### DISCUSSION TITLE
Flip left to right

### DISCUSSION THREAD

**Discussion by sudheesh-k** [DHI/mikeio#710] [General]
Hi,
Is there any method available in MIKEIO for writing data from left to right (like flipup)

Thanks for the help

**Comment by ecomodeller** [comment]
Use [`numpy.flip`](https://numpy.org/doc/stable/reference/generated/numpy.flip.html) to flip the data.

```python
import numpy as np

da.values = np.flip(da.values, axis=1)

da.to_dfs("flipped_lr.dfs2")
```

**Reply by sudheesh-k** [reply]
thanks
